src/components/Activities/Activities.jsx

```diff
@@ -47,8 +63,24 @@ const styles = theme => ({
 });
 
 class Activities extends Component {
+
+    getMessageForEventType = record => {
+        const {classes} = this.props
+        switch (record.eventType) {
+            case 'INITIATIVE_JOINED':
+                return record.actorName + ' doet lekker mee'
```

Comment:
doet mee aan de actie

Reply:
:)
It was meant as a joke. The activities part is not finished yet. Need to define all messages (ao).